camel-quarkus-snakeyaml not compatible with Quarkus 3.0.0.Alpha6

SnakeYAML has been upgraded to 2.x:

```
2023-03-15T16:52:20.6319162Z Caused by: java.lang.NoSuchMethodError: org.yaml.snakeyaml.representer.Representer: method 'void <init>()' not found
2023-03-15T16:52:20.6319978Z 	at org.apache.camel.component.snakeyaml.SnakeYAMLDataFormat.defaultRepresenter(SnakeYAMLDataFormat.java:419)
2023-03-15T16:52:20.6320884Z 	at org.apache.camel.component.snakeyaml.SnakeYAMLDataFormat.getYaml(SnakeYAMLDataFormat.java:150)
2023-03-15T16:52:20.6321994Z 	at org.apache.camel.component.snakeyaml.SnakeYAMLDataFormat.marshal(SnakeYAMLDataFormat.java:114)
2023-03-15T16:52:20.6322867Z 	at org.apache.camel.support.processor.MarshalProcessor.process(MarshalProcessor.java:64)
2023-03-15T16:52:20.6323574Z 	... 48 more
```

We need:

https://issues.apache.org/jira/browse/CAMEL-19130

I will disable the affected tests for now.

--- a/integration-tests/dataformat/pom.xml
+++ b/integration-tests/dataformat/pom.xml
@@ -67,7 +67,6 @@
 
 
     <profiles>
-        <!-- TODO: https://github.com/apache/camel-quarkus/issues/4662
         <profile>
             <id>native</id>
             <activation>
@@ -95,7 +94,6 @@
                 </plugins>
             </build>
         </profile>
-        -->
         <profile>
             <id>virtualDependencies</id>
             <activation>

--- a/integration-tests/dataformat/src/test/java/org/apache/camel/quarkus/component/dataformat/it/DataformatIT.java
+++ b/integration-tests/dataformat/src/test/java/org/apache/camel/quarkus/component/dataformat/it/DataformatIT.java
@@ -17,9 +17,7 @@
 package org.apache.camel.quarkus.component.dataformat.it;
 
 import io.quarkus.test.junit.QuarkusIntegrationTest;
-import org.junit.jupiter.api.Disabled;
 
-@Disabled("https://github.com/apache/camel-quarkus/issues/4662")
 @QuarkusIntegrationTest
 class DataformatIT extends DataformatTest {
 

--- a/integration-tests/dataformat/src/test/java/org/apache/camel/quarkus/component/dataformat/it/DataformatTest.java
+++ b/integration-tests/dataformat/src/test/java/org/apache/camel/quarkus/component/dataformat/it/DataformatTest.java
@@ -29,14 +29,12 @@ import io.quarkus.test.junit.QuarkusTest;
 import io.restassured.RestAssured;
 import org.apache.commons.io.IOUtils;
 import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.MethodSource;
 
 import static org.hamcrest.CoreMatchers.equalTo;
 
-@Disabled("https://github.com/apache/camel-quarkus/issues/4662")
 @QuarkusTest
 class DataformatTest {
 

